examples/using-gatsby-source-wordpress-next/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
💜 for including an example site alongside the plugin

Reply:
Thanks :) been using it to develop the plugin